packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/routing.js

```diff
@@ -8,11 +8,27 @@ export default /* @ngInject */ ($stateProvider) => {
     },
     resolve: {
       nodeId: /* @ngInject */ ($transition$) => $transition$.params().nodeId,
-      orderPrivateBandwidthLink: /* @ngInject */ ($state, nodeId) =>
+      handleError: /* @ngInject */ (Alerter) => (error) => {
+        Alerter.alertFromSWS(error.message, 'ERROR', 'nutanix_node_alert');
+      },
+      goBack: /* @ngInject */ ($state, $timeout) => (scrollToTop) => {
+        $state.go('^');
+
+        if (scrollToTop) {
+          $timeout(() => {
+            const element = document.getElementById('nutanix_dashboard_node');
+            if (element) {
+              element.scrollIntoView({ behavior: 'smooth', block: 'start' });
+            }
+          }, 300);
```

Comment:
**nitpick:** Instead of `$timeout`, rely on the promise returned by `$state.go()`.
```suggestion
        $state.go('^').then(() => {
           if (scrollToTop) {
              const element = document.getElementById('nutanix_dashboard_node');
              element?.scrollIntoView({ behavior: 'smooth', block: 'start' });
           }
        });
```